listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -77,20 +77,20 @@ def update_recent_listens(self, unique):
 
         # Don't take this very seriously -- if it fails, really no big deal. Let is go.
         if recent:
-            cache._r.zadd(cache._prep_key(self.RECENT_LISTENS_KEY), recent, nx=True)
+            cache._r.zadd(cache._prep_key(self.RECENT_LISTENS_KEY, cache._glob_namespace), recent, nx=True)
```

Comment:
these changes are wrong - prep_key adds the global namespace to the beginning of the key always, this argument should be None